configs/ext/README.md

```diff
@@ -1,26 +1,22 @@
 Summary
 -------
 
-The two files:
+The file:
```

Comment:
Commenting here because I cannot do it on the commit message.
The message says that you are modifying also  `configs/README.txt` file in this commit, but actually you are only changing `configs/ext/README.txt`. I think it would be nice to update it to keep it aligned with the actual commit action.

Reply:
Ah, I am missing the `configs/README.txt` file from that commit. I will add it now since it expresses how configuration files should be used.